### ISSUE TITLE
AWS Site has been changed - New fields

### ISSUE THREAD

**Issue by gshafer** [bklavet/Alexa-ArduinoYun-garagedoor-skill#2]
Got the Yun stuff working with no problems. Thanks. When I got to the Amazon AWS it seems the screens have been updated since you wrote this. I am going around in circles, not knowing how to fill in some fields. Would appreciate any direction you can give as I am sure you understand it way better than me !

**Comment by gshafer** [comment]
Managed to figure it using the information in your files. All working great now. Thanks for the project, just what I need to raise and lower my blinds.

**Comment by bklavet** [comment]
Cool, glad you were able to sort out A.W.S.

What kind of motor did you use for the blinds?

**Comment by gshafer** [comment]
It is a commercial product made by Somfy.  I have motorized blinds on 3 windows that I put down at night and open in the morning. I modified your code to have 3 options: up, down and pause. I used a Yun with a 4 relay hat, and each option triggers a separate relay. I then wired the relay across an existing remote’s push buttons. So by saying “Alexa ask the blinds to go up” , the blinds all open.

Thanks for posting this great project. I have been thinking about how to do this for a while and with your project I was able to get it to work.
